cmd/kueuectl/app/list/list_pods.go

```diff
@@ -202,6 +202,7 @@ func (o *PodOptions) getForObjectInfos(clientGetter util.ClientGetter) ([]*resou
 		Do()
 
 	if r == nil {
+		//nolint:staticcheck // This is a user-facing error, so we keep it as is.
```

Comment:
What is it proposing to change?

Reply:
In Go, error messages should start with a lowercase letter. But this is user-facing error. So just ignore it.

Reply:
This is probably about lowercasing the first word. I would prefer to actually lowercase all the errors in kueuectl. 

Can you show example output from kueuectl for one of the errors? 

If we would start with lowercase we could probably prefix it generically with "Error: invalid object ... ". It might be a follow up though indeed.

Reply:
Actually, we have `Error:` prefix.

```
Error: Error building client for: Job/sample-job-ctkvp
```

Reply:
Fixed

Reply:
Also fixed other places.